bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
It is, but the ExternalArrayBuffer is cleaned up before this. I'll rescope this fix properly.